HACKING

```diff
@@ -341,6 +341,22 @@ Example::
     {
         ...
 
+Translation
+^^^^^^^^^^^
+
+Strings which are visible to geany users need to be translated and thus
+should be enclosed in ``_( )`` to have them being recognized by the GNU
+translation tools. Also spaces right after the first ``"`` of a string
```

Comment:
`_()` is actually a function call, where `_` is an alias for `gettext`.  But yes, it's also used to recognize strings that need translating by the tools that read the source code for extracting strings for that purpose.
But the formulation you chose is not perfect, it's not only for recognizing the string, but also to actually translate it for the program. `gettext()` (for which `_` is an alias) is returning the translated version of the passed-in string.